server/sonar-server/src/main/java/org/sonar/server/qualityprofile/QProfileFactoryImpl.java

```diff
@@ -127,6 +127,10 @@ public void delete(DbSession dbSession, Collection<QProfileDto> profiles) {
     db.defaultQProfileDao().deleteByQProfileUuids(dbSession, uuids);
     db.qualityProfileDao().deleteOrgQProfilesByUuids(dbSession, uuids);
 
+    // Permissions are only available on custom profiles
```

Comment:
outside the scope of this PR: do you know why a factory class has a public delete method?

Reply:
It should be refactored later indeed